File: SampleSetup/Output/PHP/SampleProject/Protocol/Body/ApiTokenPostArg.php
<?php
namespace SampleProject\Protocol\Body;

class ApiTokenPostArg {
    /**
     * @deprecated Message: Use the other one
     * @var string
     */
    public $UserName;

    /// <summary>
    /// A sample summary.
    /// </summary>
    /**
     * A sample summary.
     * @var int
     */
    public $UserNameKind;

    /// <summary>
    /// A multiline summary.
    /// test explanations :)
    /// </summary>
    /**
     * A multiline summary.
     * test explanations :)
     * @var string
     */
    public $Password;

    public function __construct()
    {
        $this->UserName = "";
        $this->UserNameKind = 0;
        $this->Password = "";
    }
}

File: SampleSetup/Output/Kotlin/SampleProject/Protocol/Body/ApiTokenPostArg.kt
package SampleProject.Protocol.Body

open class ApiTokenPostArg {
    @Deprecated(message = "Use the other one")
    var UserName: String = ""

    /**
     * A sample summary.
     */
    var UserNameKind: Int = 0

    /**
     * A multiline summary.
     * test explanations :)
     */
    var Password: String = ""
}
